homeassistant/components/freebox/config_flow.py

```diff
@@ -69,7 +69,17 @@ async def async_step_link(
 
             # Check permissions
             await fbx.system.get_config()
-            await fbx.lan.get_hosts_list()
+            try:
+                # This function is not available if freebox is configured in bridge mode
+                await fbx.lan.get_hosts_list()
+            except HttpRequestError as error:
+                _LOGGER.debug("The error: %s", error)
+                if "APIResponse" in str(error) and '"error_code": "nodev"' in str(
```

Comment:
Why are you checking here differently than in #103221?
It looks like it is the same error, correct?

If yes we should create a common function to check if we are in bridge mode or not